maple-sdk/usage-guide.md

```diff
@@ -0,0 +1,64 @@
+# Usage Guide
+
+This comprehensive guide will help you understand how to use the Maple SDK for various tasks, such as connecting to contracts and executing transactions.
+
+## Overview
+
+The Maple SDK simplifies interacting with Maple Finance's smart contracts on Ethereum blockchain. It includes address mappings for each contract to facilitate deploying to different networks: `Ethereum Mainnet` and `Goerli`. The available networks are `mainnet-prod`, `mainnet-dev`, `goerli-prod` and `goerli-dev`.
+
+Access addresses from the `addresses` object exported from `maple-js`. See a list of available contracts in `src/addresses/*.ts`.
+
+## Getting Started
+
+**Accessing Contract Addresses**
+
+Access contract addresses using the `addresses` object exported from `maple-js`. You can find the list of available contracts in the `src/addresses/*.ts` files.
+
+**Code Example:**
+
+```js
+import { mapleGlobals } from '@maplelabs/maple-js';
+
+const contractAddress = addresses['mainnet-prod'].MapleToken;
+const signer = 'yourSigner';
+
+const contract = mapleGlobals.core.connect(contractAddress, signer);
+```
+
+## Connecting to a Contract
+
+To connect to a contract, you will need a contract address and a signer (usually a wallet instance). For more information on creating a signer, refer to the [ethers documentation](https://docs.ethers.io/v5/) or your preferred web3 library.
```

Comment:
Is it possible to construct a signer using for example `web3.js` and pass it in when connecting? I don't know that that is the case (we reference "web3" library)?